tests/test_base.py

```diff
@@ -10,11 +10,13 @@
 from pytest import raises, approx
 from voluptuous import Error, Invalid, truth
 import mitxgraders
-from mitxgraders import ListGrader, StringGrader, ConfigError, FormulaGrader, CalcError, __version__
+from mitxgraders import (ListGrader, StringGrader, ConfigError, FormulaGrader,
+                         CalcError, RandomFunction, __version__)
 from mitxgraders.helpers import validatorfuncs
-from mitxgraders.helpers.calc import evaluator, UnableToParse, UndefinedVariable
```

Comment:
A bunch of stuff in `test_base.py` testing stuff from mitxgraders.helpers. Maybe we should just make `test/helpers/test_calc.py` and `test/helpers/test_validatorfuncs.py`?

Reply:
I've refactored it out as suggested.